### ISSUE TITLE
[Bug] Value -<large_int> of does not fit into uint128 ("chia\wallet\wallet_state_manager.py", line 551, in _get_unconfirmed_balance)

### ISSUE THREAD

**Issue by andriitalkstech** [Chia-Network/chia-blockchain#10349]
### What happened?

Python exception in the wallet

### Version

1.3 Beta

### What platform are you using?

Windows

### What ui mode are you using?

GUI

### Relevant log output

```shell
2022-02-21T11:44:49.423 wallet chia.rpc.rpc_server        : WARNING  Error while handling message: Traceback (most recent call last):
  File "chia\rpc\rpc_server.py", line 215, in safe_handle
  File "chia\rpc\rpc_server.py", line 206, in ws_api
  File "chia\rpc\wallet_rpc_api.py", line 627, in get_wallet_balance
  File "chia\wallet\wallet.py", line 114, in get_unconfirmed_balance
  File "chia\wallet\wallet_state_manager.py", line 530, in get_unconfirmed_balance
  File "chia\wallet\wallet_state_manager.py", line 551, in _get_unconfirmed_balance
  File "chia\util\ints.py", line 42, in __new__
ValueError: Value -<NUMBER> of does not fit into uint128
```

**Comment by janit** [comment]
I have the same issue, but on CLI:

```
(venv) janit@informer:~/chia-blockchain$ chia wallet show
Wallet height: NNN
Sync status: Not synced
Balances, fingerprint: NNN
Exception from 'wallet' {'error': 'Value -NNN of does not fit into uint128', 'success': False}
```

I can give NNN details in private if needed.

**Comment by esaung** [comment]
known issue, we are working on it

**Comment by emlowe** [comment]
We believe all such issues are fixed in 1.3 release - please re-open if you have any problems with 1.3
